controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -24,16 +24,22 @@ def recording_path() -> Path:
     date = now.date().isoformat()
     # Windows doesn't like colons in filenames
     name = now.isoformat().replace(':', '-')
-    return Path(date) / name / '{}.html'.format(name)
+    return Path(date) / name / '{}'.format(name)
 
 
 @contextlib.contextmanager
 def record_animation(supervisor: Supervisor, file_path: Path) -> Iterator[None]:
     file_path.parent.mkdir(parents=True, exist_ok=True)
-    print("Saving animation to {}".format(file_path))
-    supervisor.animationStartRecording(str(file_path))
+    print("Saving animation to {}.html".format(file_path))
+    print("Saving video to {}.mp4".format(file_path))
+    supervisor.animationStartRecording(str(file_path) + '.html')
+    supervisor.movieStartRecording(str(file_path) + '.mp4',
+                                   width=1920, height=1080,
+                                   quality=100, codec=0, 
+                                   acceleration=1, caption=False)
```

Comment:
nit: please could we wrap this to avoid this mixed wrapping style? The local style is like:
```suggestion
    supervisor.movieStartRecording(
        str(file_path) + '.mp4',
        width=1920, 
        height=1080,
        quality=100, 
        codec=0, 
        acceleration=1, 
        caption=False,
    ) 
```